packages/server/src/validator/objectTypes.ts

```diff
@@ -1,109 +1,140 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
 import { Validator } from '.';
 import { predefined } from '@hashgraph/json-rpc-relay';
+import { IObjectSchema, IObjectValidation } from '../types/validator/objectTypes';
 
-export const OBJECTS_VALIDATIONS = {
+export const OBJECTS_VALIDATIONS: { [key: string]: IObjectSchema } = {
   blockHashObject: {
-    blockHash: {
-      type: 'blockHash',
-      nullable: false,
+    properties: {
```

Comment:
QQ: why do we want to wrap these in an extra nested object?

Reply:
Because we want to also have the ability to configure validations for the root object (e.g., `failOnEmpty` or `failOnUnknownProperties`)

Reply:
fair thanks for confirming!